tuna.go

```diff
@@ -347,11 +374,15 @@ func (c *Common) CreateServerConn(force bool) error {
 					continue RandomSubscriber
 				}
 
-				if entryToExitPrice > c.EntryToExitMaxPrice {
+				entryToExitMaxPrice, exitToEntryMaxPrice, err := ParsePrice(c.EntryServiceInfo.MaxPrice)
```

Comment:
It's better to parse max price in the beginning of the function so no need to parse price for many times